crates/utils/re_log/src/setup.rs

```diff
@@ -156,6 +156,8 @@ impl log::Log for PanicOnWarn {
         }
     }
 
+    // That must be a single place where panicing is allowed.
```

Comment:
practically that comment won't hold. It's also not helpful in _why_ it's allowed here which is much much more interesting!

Reply:
Should be clear now as I included the reason why it's expected and that `panic!` isn't allowed elsewhere.